pages/posts/w3c-wot-case-study.md

```diff
@@ -0,0 +1,137 @@
+
+Adopting JSON Schema means that the W3C Web of Things does not need to invest
+effort on inventing another schema language. As a consequence of the popularity
+of JSON Schema, existing tooling can be often reused instead of implementing
+custom parsers, validators, code generators and UI generators for all the
+programming languages that the community might be interested in.
+
+JSON Schema has proven to be a foundational block for creating higher-level
+tooling that is specific to the W3C Web of Things. As a notable example, the
+community has developed a tool that makes use of JSON Schema definitions inside
+Thing Descriptions to generate matching payloads for the purpose of stress and
+penetration testing.
+
+Given the growing popularity of using JSON Schema to generate web-based forms,
+W3C Web of Things specifications can be used to automatically generate
+dashboards and user interfaces to interact with the objects modeled by Thing
+Descriptions.
+
+## Key Impact Results
+
+The ultimate metric of success for a standard like the W3C Web of Things is
+adoption, and community adoption is not something you can just "engineer".
+"Because we adopt JSON Schema, developers who are familiar with it have a
+smaller learning curve when learning the W3C Web of Things standard." - Ege
+Korkan
+
+## W3C Web of Things
+
+World Wide Web Consortium (W3C) is the standardization organization behind most
+of the standards used in the Web. We are the Web of Things Working and Interest
+Groups, who work on the standardization of Web of Things at the W3C.
+
+The Web of Things groups aim to create an interoperability layer so that
+different Internet of Things platforms, protocols and standards can operate
+together.
+
+The work has started as a discussion in the Web of Things Community Group in
+2013-2014.  It has proceed to an Interest Group in 2015 that has collected the
+use cases and defined the standards to be worked on. Since 2016, the working
+group is working on different standards on the Web of Things with the first
+publications of the Thing Description and Architecture recommendations in 2019.
+
+At the time of this writing, the Working Group consists of 96 participants
+representing 36 organizations and the the Interest Group consists of 131
+participants representing 48 organizations. 
+
+![W3C Web of Things Examples](/blog/img/posts/2022/w3c-wot-case-study/wot-examples.webp)
+
+## Getting Started
+
+We invite you to leverage your existing JSON Schema knowledge to play with the
+Internet of Things through the use of the W3C Web of Things specifications and
+tools. You might already have a smart device next to you that does not come
+with a Thing Description!
+
+If you write your own Thing Description, you can programatically interact with
+your device by using the [Node.js Web of Things
+implementation](https://github.com/eclipse/thingweb.node-wot), or play with it
+through the [Node-RED](https://www.youtube.com/watch?v=oAcYbJ6P9bU)
```

Comment:
you can also link to the node-red web of things plugin (based on node-wot): https://www.npmjs.com/package/node-red-contrib-web-of-things